library/java/net/openid/appauth/AppAuthConfiguration.java

```diff
@@ -94,12 +114,33 @@ public Builder setConnectionBuilder(@NonNull ConnectionBuilder connectionBuilder
             return this;
         }
 
+        /**
+         * Disables https validation for the issuer identifier.
```

Comment:
Perhaps this should be documented/log a big warning that this is very unsafe? That way people don't disable it without being sure it's the right thing they want to do.